### ISSUE TITLE
Finding the "best" line

### ISSUE THREAD

**Issue by oldoc63** [oldoc63/learningDS#467]

**Comment by oldoc63** [comment]
In this plot, we see two points on either side of the line. One of the points is one unit below the line (labeled -1). The other point is three units above the line (labeled 3). The total squared error (loss) is: $loss = (-1)^2+(3)^2=1+9=10$

**Comment by oldoc63** [comment]
https://content.codecademy.com/programs/data-science-path/line-fitter/line-fitter.html

**Comment by oldoc63** [comment]
The interactive visualization in the browser lets you try to find the line of best fit for a random set of data points:
- The slider on the left controls the `m` (slope)
- The slider on the right controls the `b` (intercept)

You can see the total loss on the right side of the visualization. To get the line of best fit, we want this loss to be as small as posible.

To check if you got the best line, check the "Plot-Best-Fit" box.

Randomize a new set of points and try to fit a new line by entering the number of points you want (try 8!) and pressing `Randomize Points`.
